README.md

```diff
@@ -1,199 +1,263 @@
 Python for OpenShift - Docker images
-========================================
+====================================
 
 This repository contains sources of the images for building various versions
 of Python applications as reproducible Docker images using
-[source-to-image](https://github.com/openshift/source-to-image).
-User can choose between RHEL and CentOS based builder images.
+[Source-to-Image](https://github.com/openshift/source-to-image).
+User can choose between CentOS and RHEL based builder images.
 The resulting image can be run using [Docker](http://docker.io).
 
 
 Versions
----------------
-Python versions currently provided are:
-* python-3.3
+--------
 
-RHEL versions currently supported are:
-* RHEL7
+Python versions currently provided:
+* Python 3.3
 
-CentOS versions currently supported are:
-* CentOS7
+CentOS versions currently supported:
+* CentOS 7
 
+RHEL versions currently supported:
+* RHEL 7
 
-Installation
----------------
-To build Python image, choose between CentOS or RHEL based image:
-*  **RHEL based image**
 
-    To build a rhel-based python-3.3 image, you need to run the build on a properly
-    subscribed RHEL machine.
+Image name structure
+--------------------
 
-    ```
-    $ git clone https://github.com/openshift/sti-python.git
-    $ cd sti-python
-    $ make build TARGET=rhel7 VERSION=3.3
-    ```
+Structure: **openshift/1-2-3**
+
+1. Platform name - python
+2. Platform version (without dots)
+3. Base builder image - centos7/rhel7
+
+Examples: `openshift/python-33-centos7`, `openshift/python-33-rhel7`
+
+
+Installation
+------------
+
+To build a Python image, choose between CentOS or RHEL:
 
 *  **CentOS based image**
 
-    This image is available on DockerHub. To download it use:
+  This image is available on DockerHub. To download it use:
 
-    ```
-    $ docker pull openshift/python-33-centos7
-    ```
+  ```
+  $ docker pull openshift/python-33-centos7
+  ```
 
-    To build Python image from scratch use:
+  To build it from scratch:
 
-    ```
-    $ git clone https://github.com/openshift/sti-python.git
-    $ cd sti-python
-    $ make build VERSION=3.3
-    ```
+  ```
+  $ git clone https://github.com/openshift/sti-python.git
+  $ cd sti-python
+  $ make build VERSION=3.3
+  ```
+
+*  **RHEL based image**
 
-**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+  To build this image you need to run the build on a properly subscribed RHEL
+  machine:
+
+  ```
+  $ git clone https://github.com/openshift/sti-python.git
+  $ cd sti-python
+  $ make build TARGET=rhel7 VERSION=3.3
+  ```
+
+Notice: By omitting the `VERSION` parameter, the build action will be performed
```

Comment:
Leave noticed bolded, unless you'll change all the images. We diverged the descriptions once, I've spent quite some time to have them in common, so I'll be picky about it.